src/components/post/post.js

```diff
@@ -1,28 +1,71 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './post.css';
 import '../../styles.css';
 import avatar from '../../assets/defaultImg.png';
+import axios from 'axios';
+import { useAuth } from '../../context/authContext';
```

Comment:
put all the context related part of code into an index.js file and import it from there to make it more readable

Reply:
understood, thanks for this suggestion will implement it.